pyproject.toml

```diff
@@ -988,7 +988,6 @@ module = [
     "sentry.monitors.endpoints.organization_monitor_checkin_attachment",
     "sentry.monitors.endpoints.organization_monitor_details",
     "sentry.monitors.endpoints.organization_monitor_index",
-    "sentry.monitors.endpoints.organization_monitor_stats",
```

Comment:
oh why did it get removed?

Reply:
I did an oopsie -- https://github.com/getsentry/sentry/pull/50603